.github/workflows/presubmit.yml

```diff
@@ -23,6 +24,7 @@ jobs:
           - os: ubuntu-20.04
             mainmatrix: false
             arch: aarch64
+            debug: 1
```

Comment:
What does this do?

I'm not conversant with the CI but how does this change stay scoped to just the Debug build?

Reply:
I'm not sure I really understand the question, but I will try my best to explain.

There are five builds specified here:
* ubuntu-20.04 (x86-64)
* ubuntu-20.04 arm
* ubuntu-20.04 aarch64
* macos-latest
* windows-latest

The "include" section is what causes the CTS to be built with additional architectures for ubuntu-20.04.

"debug: 1" only applies to ubuntu-20.04 with the arch being "aarch64".

The "ubuntu-20.04 arm" and "ubuntu-20.04" (x86-64) builds do not contain "debug: 1" in their configuration.  In these cases, JOB_ENABLE_DEBUG will not be "1".  (We could perhaps add a default of 0, but there is no strict need, and the same would apply for gl - so it seems best to keep them consistent for now.)

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-expanding-configurations for documentation regarding workflows in GitHub Actions.

Reply:
Thanks you, completely clear now - I should have looked more closely and rtfm.

So in summary we only do a debug build on ubuntu-20.04-aarch64.

This does mean - if I understand correctly - that we have not just added a debug build option, but changed the ubuntu-20.04-aarch64 from release to debug?

Sorry to be a pedant, and I can be :-), but should the commit message not say something along Add debug build option and change ubuntu-20.04-aarch64 from Release to Debug?

Reply:
That would have been the case, except we've discovered that we have _only_ doing debug builds so far, and that we should include release builds.

"Add debug build option and change ubuntu-20.04-aarch64 from Release to Debug?" might have been a little on the verbose side.  I will go with "Include release builds in GitHub Actions" as the commit title and elaborate below, unless there's a more accurate title I could use that is also concise.